lib/stm32/common/rng_common_v1.c

```diff
@@ -94,10 +94,16 @@ uint32_t rng_get_random_blocking(void)
         uint32_t rv;
         bool done;
         do {
-                if (RNG_SR & RNG_SR_SECS) {
-                        rng_disable();
-                        rng_enable();
+
+                if (RNG_SR & RNG_SR_SEIS) {
+                        RNG_SR = RNG_SR & ~RNG_SR_SEIS;
+                        for (int i = 12; i != 0; i--) {
+                                rv = RNG_DR;
+                        }
+                        RNG_CR &= ~RNG_CR_RNGEN;
+                        RNG_CR |= RNG_CR_RNGEN;
```

Comment:
My only concern with this series is this one.  I'm really not sure that "both" is a good compromise.  I kinda feel like if you do the disable/enable, there was no point in the readout, and the readout of 12 is in the g0 manual as a speed up?  _or_ it's a different implementation? (more doubtful)  I just... feel weird doing both.

Reply:
eh, I'm not using the peripheral.  I'm going to go with your judgement on this.